### DISCUSSION TITLE
Issue with Login Due to Missing Google Authenticator Codes

### DISCUSSION THREAD

**Discussion by CodeMA123** [community/community#145945] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### GitHub Feature Area

Issues

### Body

I am experiencing an issue accessing my GitHub account. I recently reset both my laptop and phone, and in the process, I lost the codes from my Google Authenticator app, couldn't find recovery codes also.

When I attempt to recover my account, I reach the step where it asks me to verify an alternative factor. However, the process is getting stuck with the following message:

“Next, we need to verify an alternative factor. This can be a verified device, an SSH key, or a personal access token associated with this account.”

Unfortunately, I am unable to proceed as it states:

No valid SSH keys associated with the account.
No valid personal access tokens available.
As a result, I am unable to verify my device or perform the recovery process.

Could you please guide me on how to recover my account or provide alternative options to regain access, I need this very much since all repos and commit history needed for me to secure a job for me, I'm a Sri Lankan, Please please consider my request. I would appreciate any assistance you can provide.

Thank you for your support!

Best regards,
Thakshila.

GitHub username: 
ThakshilaJayathilake

**Comment by jamesbrown941** [comment]
Contact GitHub Support: Use [GitHub Support](https://support.github.com/contact) to explain your issue with as much detail as possible.
Search for Recovery Codes: Double-check old backups or emails for stored codes.
Provide Backup Info: Mention any linked emails or other details to help verify your account.
GitHub Support is your best option—good luck!

**Reply by ThakshilaJayathilake** [reply]
Finally found recovery codes from my backup. Thanks

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @CodeMA123 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by tyagraj777** [comment]
Hello @CodeMA123 ,

Thanks for posting question, it can be really upsetting situation.. 

I'm sorry to hear about the difficulties you're facing with your GitHub account recovery. Losing access to your two-factor authentication (2FA) codes and recovery options can be challenging. While I cannot directly assist with account recovery, here are the steps you should follow to try and regain access to your account:

1] **Submit a GitHub Support Request**

1. Visit the [GitHub Support contact page](https://support.github.com/contact).
2. Select "I need help with account recovery" or a similar option.
3. Fill out the form with as much detail as possible, including:

 - Your GitHub username (ThakshilaJayathilake).
 - The email address associated with your account.
 - A description of the issue (include details like your situation, previous devices, and the 2FA reset issue).

4. If you have any old emails from GitHub confirming account actions or repo-related emails, attach them to prove ownership.


2] **Prove Ownership of the Account**
GitHub will ask for verification to ensure you are the rightful owner of the account. This could include:

- **Billing Information**: If you have ever made payments (e.g., for GitHub Pro), provide receipts or transaction details.
- **Email Records**: Screenshots or forwarded copies of emails from GitHub related to the account.
- **Repository Details**: Specific details about your repositories, including:

 - Repository names.
 - Last commit SHA or recent activity.
 - Any collaborators or forks of your repositories.

3] Wait for Support Response
Account recovery can take time because GitHub manually verifies ownership for security reasons. Be patient and monitor your email for responses.

Note:
to avoid similar situation in future, _Backup 2FA recovery codes_, _Add multiple 2FA methods_, _Enable SSH Keys and Personal Access Tokens_

Hope your account recovers soon !!!

**Comment by naziopa69** [comment]
Google.com

**Reply by naziopa69** [reply]
https://Google.com

**Comment by MasterInQuestion** [comment]
There seems to be constant 2FA lockout issues...
    Regardless unsalvageable:
    https://github.com/orgs/community/discussions/63813#discussioncomment-10963614

**Comment by ThakshilaJayathilake** [comment]
Thank you so much. I recovered my account. I appreciate all of your support & they were helpful. @MasterInQuestion  @jamesbrown941 , @tyagraj777   @queenofcorgis

**Reply by tyagraj777** [reply]
Select any relevant answer, help others too .. !!! Njoy

**Reply by MasterInQuestion** [reply]
Luckily you found one of the credentials.
    Else afraid...
